readme.md

```diff
@@ -77,16 +77,16 @@ The app config should then have a common logging section like below. Be sure to
 ```
 
 ## Application Insights Quickstart
-There are different packages for each major Application Insights version. Install the correct package for your referenced Application Insights version. This example installs the adapter for Application Insights v0.17:
+There are different packages for each major Application Insights version. Install the correct package for your referenced Application Insights version. This example installs the adapter for Application Insights v2.4.0:
 
-    PM> Install-Package Microsoft.ApplicationInsights -Version 0.17.0
+    PM> Install-Package Microsoft.ApplicationInsights -Version 2.4.0
```

Comment:
The correct way to install the package is actually this:
PM> Install-Package Common.Logging.ApplicationInsights240